addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml

```diff
@@ -23,11 +24,11 @@
         <Key android:codes="d" android:popupCharacters="#%đďḏ" ask:hintLabel="# %"/>
         <Key android:codes="f" android:popupCharacters="^\u0026" ask:hintLabel="^ \u0026"/>
         <Key android:codes="g" android:popupCharacters="`°ĝ" ask:hintLabel="` °"/>
-        <Key android:codes="h" android:popupCharacters="_~ĥḥḫẖ" ask:hintLabel="_ ~"/>
+        <Key android:codes="h" android:popupCharacters="\u005f~" ask:hintLabel="\u005f~"/>
```

Comment:
what is this change? the popup of `h` is `f`?

Reply:
No, thats unicode. it's _ and ~. For some reasons "_~ĥḥḫẖ" would not work. You'd never get _ and ~ as first suggestions on long press.

Reply:
the reason I removed the ĥḥḫẖ, was because I couldn't get  the underscore and the ~ as first choices. No matter what I tried, the only way was to  remove the "ĥḥḫẖ".
Here what happened when you long pressed on h:
<img src="https://camo.githubusercontent.com/..." data-canonical-src="https://user-images.githubusercontent.com/16888339/77008762-100c2200-6967-11ea-971f-3fb7d379e638.jpg" width="400" height="308" />

Reply:
I'm confused.. what is the first image and what is the second? I don't want to lose  `ĥḥḫẖ` in the long-press popup. If this is really broken, there is another option to use `android:popupKeyboard`. See https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/languages/english/pack/src/main/res/xml/qwerty.xml#L24 for an example

Reply:
Thanks! I'll have a look at that.
The first picture is ask master. The second picture is my branch.

Reply:
Done! I fixed it with your advice